test/unit/controller/payment-links/get-start.controller.test.js

```diff
@@ -0,0 +1,163 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+const { expect } = require('chai')
+const mockResponses = {}
+
+const getController = function (mockResponses) {
+  return proxyquire('../../../../app/controllers/payment-links/get-start.controller', {
+    '../../utils/response': mockResponses
+  })
+}
+
+describe('The Worldpay MOTO account warning', () => {
+  let req
+  let res
+
+  describe('when the gateway account has a Worldpay credentials object with a merchant code ending with ‘MOTO’', () => {
+    before(() => {
```

Comment:
(not blocking) Any reason each test has its own setup (`before()`) method instead of a global one? 

We could have a common setup and within tests, `req` object values can be set as per the test.

Reply:
Will move some stuff into a global `beforeEach(…)` function.